client/src/cmdhficlass.c

```diff
@@ -1049,74 +1010,82 @@ static int CmdHFiClassEView(const char *Cmd) {
     free(dump);
     return PM3_SUCCESS;
 }
-
 static int CmdHFiClassDecrypt(const char *Cmd) {
+    CLIParserContext *clictx;
+    CLIParserInit(&clictx, "hf iclass decrypt",
+                  "3DES decrypt data\n"
+                  "This is naive implementation, it tries to decrypt every block after block 6.\n"
```

Comment:
The spelling mistake was before but shouldn't it be  "This is a naive" ?

Reply:
Yup, I'll fix that!